src/modules/test/partTwo.test.js

```diff
@@ -0,0 +1,109 @@
+/* eslint-disable */ 
```

Comment:
- [ ] Kindly ensure you dont disable any of the linters as linters helps to reduce bugs in your codebase and also focus you to write workable code. Please remove the linters you disabled